MAUI/Kanban-Board/Getting-Started.md

```diff
@@ -505,7 +509,415 @@ doneColumn.Categories = new List<object>() { "Done" };
 
 {% endtabs %}
 
-### Running the Application
+## Custom Model mapping
+
+You can also map custom model data to our Kanban.
+
+### Define the Custom Model
+
+Create a custom model class as shown below.
+
+{% highlight C# %}
+
+public class KanbanCustomModel : INotifyPropertyChanged
```

Comment:
Follow this

![Uploading image.png…]()

Reply:
Code snippet template changed as above